packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.stories.tsx

```diff
@@ -57,19 +56,5 @@ export default {
 };
 
 export const Default = {
-  render: () => <Story placeholder='Enter text...' />,
-};
-
-export const Text = {
   render: () => <Story text={initialText} placeholder='Enter text...' />,
 };
-
-export const Automerge = {
-  render: () => <Story text={initialText} placeholder='Enter text...' automerge />,
-};
-
-export const Listener = {
```

Comment:
we should recreate this story somewhere, it's helpful for validating the listener is working as expected. I needed it when getting the local files plugin working again